### PR TITLE
Add basic Aurora UI prototype

### PR DESCRIPTION
**Pull request by Flaque**
Add a really simple UI for adding notes and a feed to see them on. 
screen shot 2017-09-10 at 2 03 03 pm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,11 +40,39 @@ You can test the application with `npm run test` which will run [jest](https://f
...
+    expect(tree).toMatchSnapshot();
+  });
+```
+
+When you run your tests, if a snapshot has been changed it will alert you. If you meant to do this, then you can update automagically it by pressing "u". (It will have a prompt you can follow). 
+
+If you didn't mean to do this, then go fix your things.
+
+You can learn more about snapshot testing here. 
+
```
Comment (kmccrohan) [inline README.md]: Does this still work well for elements that have some content created dynamically that might be inconsistent?

File: packages/aurora-editor/test/CardAtBottomEditor.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+describe("CardAtBottomEditor", () => {
+  it("is exported and exists", () => {
+    expect(CardAtBottomEditor).toBeDefined();
+  });
+
+  it("is a valid React element", () => {
+    const element = (
+      <CardAtBottomEditor onSubmit={() => {}} onChange={() => {}} />
+    );
+    expect(React.isValidElement(element)).toBe(true);
```
Comment (kmccrohan) [inline packages/aurora-editor/test/CardAtBottomEditor.test.js]: What does this test do exactly?

File: README.md (hunk 1 above)
Comment (Flaque) [reply README.md]: Assuming you give the React component the same inputs/props each time, yes.  React components are stateless and should have no side effects. So if, in your test, you give it:

```js 
const element = <MyReactComponent dynamicStuff={SameEachTest} />
```

You will always get the same snapshot unless you change the `MyReactComponent`.

File: packages/aurora-editor/test/CardAtBottomEditor.test.js (hunk 2 above)
Comment (Flaque) [reply packages/aurora-editor/test/CardAtBottomEditor.test.js]: I was having bugs where some things weren't being exported and rendered correctly as subclasses of React Components, so I put this in here so it wouldn't happen again. 

Basically it just checks if the item is rendering without any errors.

**Comment by Flaque** [comment]
> With prototyping UIs, is our project structured so that you can create different UIs that still leverage similar elements? And just run them with a different run command?

Yeah, that's the basic idea. React let's us do that pretty easily. You can pop an `Editor` anywhere or use a `Card` wherever you want. In general, the longer and more specific the name, the less reusable it is. 

So `Editor` is fairly generic and can be used anywhere. But `CardAtBottomEditor` is the specific editor that appears at the bottom of the screen.

**Comment by Flaque** [comment]
screen shot 2017-09-10 at 6 21 24 pm

**Pull request merged**